modules/oe_paragraphs_timeline/oe_paragraphs_timeline.module

```diff
@@ -26,88 +27,47 @@ function oe_paragraphs_timeline_field_widget_timeline_widget_form_alter(&$elemen
 
   // We can't use the protected isDefaultValueWidget() method.
   $is_default_value_widget = (bool) $form_state->get('default_value_widget');
-  if ($is_default_value_widget) {
-    return;
-  }
 
-  $element['label']['#required'] = $element['#required'];
-  $element['title']['#required'] = $element['#required'];
```

Comment:
Why this part was removed? Now the fields are not marked as required anymore for the first delta of the field.

Reply:
I've restored it.